packages/Operators/src/PointCloud/DTK_NodeToNodeOperator_impl.hpp

```diff
@@ -103,26 +103,14 @@ void NodeToNodeOperator<DIM>::setupImpl(
 	domain_iterator =
 	    domain_space->entitySet()->entityIterator( 0, domain_predicate );
     }
```

Comment:
couldn't that block that gets the iterator move into the ``getNodeCoordsAndIds`` function?